examples/src/main/java/kafka/examples/KafkaConsumerProducerDemo.java

```diff
@@ -18,17 +18,22 @@
 
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
```

Comment:
Here we us java TimeoutException, which is checked. In `KafkaExactlyOnceDemo`, we use the kafka TimeoutException, which is unchecked. Can we be consistent?